test/address_prop.ml

```diff
@@ -0,0 +1,38 @@
+open QCheck2.Gen
+open Generator.Test
+
+module SUT = Pollinate.Address
+
+let create =
+  QCheck2.Test.make ~count:1000
+    ~name:"create from string and port is successful" (pair string int)
+    (fun (address, port) ->
+      let addr = SUT.create address port in
+      addr.port == port && addr.address == address)
+
+let from_sockaddr =
+  QCheck2.Test.make ~count:1000
+    ~name:"from_sockaddr from any sockaddress is successful" sockaddress_gen
```

Comment:
Can we make it more obvious where sockaddress_gen is defined by specifying the module it's from or opening it at the beginning of this function?

Reply:
`sockaddress_gen` is coming from `Generator.Test` module, which defines all the generators needed on our suite tests.
Do you want me to remove the `open Generator.Test` at the beginning and here use `Generator.Test.sockaddress_gen` instead?
It's ok for me, so as you wish 😄

Reply:
Done, also renamed to `Generators` to be more consistent with other modules.